package.json

```diff
@@ -88,7 +88,7 @@
     "sync": "svelte-kit sync",
     "check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
-    "test": "npm run test:unit && npm run test:browser",
+    "test": "TZ=UTC vitest run",
```

Comment:
If these are split apart, should update the `.github/workflows/test.yml` to run the playwright test step.

Reply:
Playwright isn't actually running on the `sveltekit` branch. It's all aimed at the current site right now, so it's not doing anything for us.